### DISCUSSION TITLE
MissionControl 0.6.4

### DISCUSSION THREAD

**Discussion by ndeadly** [ndeadly/MissionControl#423] [General]
This is the thirteenth official release of Mission Control.

This adds full support for HOS 13.2.1, along with a few minor bugfixes. See below for the full changelog.

New users landing on this release page should first check out the readme on the [main project page](https://github.com/ndeadly/MissionControl#readme) for the official project documentation. There you can find installation and usage instructions along with an FAQ section that will answer most of your questions.

***Atmosphère >= 1.2.5 is required to run this release.***

### Notice for users of Xbox One controllers
If you have updated to the new bluetooth LE controller firmware and lost support for your controller, it is now possible to downgrade back to a compatible firmware using an Xbox One console. See [here ](https://answers.microsoft.com/en-us/xbox/forum/all/revertrollbackdowngrade-xbox-controller-firmware/25f07169-3244-4fe0-838c-949951b7c109?auth=1) for details.

(thanks to @impeeza and SpiderLink for bringing this to my attention)

### Changelog:
- Added bluetooth ips patches for 13.2.1.
- Fixed incorrect mapping of A/B buttons on Hyperkin Scout controller.
- Fixed disconnection issues affecting Xiami Mi, Steelseries Free and PowerA Moga controllers.
- Fixed a regression that caused the enable_rumble setting in the module config ini file to have no effect.
- Fixed a compilation error when building with latest devkitA64 update

<hr /><em>This discussion was created from the release <a href='https://github.com/ndeadly/MissionControl/releases/tag/v0.6.4'>MissionControl 0.6.4</a>.</em>

**Comment by motorstormm** [comment]
Hi,
I have update my Xbox One controller (not the one from Xbox series) with my Xbox series console, and now Missioncontrol 0.6.3 and 0.6.4, and my dongle Magic NS can't appear it. 
Any solution ?

**Reply by ndeadly** [reply]
There's a link to an article with instructions on downgrading in the release notes you likely joined this discussion from. As far as I know it can only be done from an Xbox One console though.

**Reply by impeeza** [reply]
Snap, i didn't realize it's a discussion. Think was a bug report 🤣

**Comment by impeeza** [comment]
Please go to a support/help site this ia for bug reports.

Saying that: Revert the firmware

**Reply by ndeadly** [reply]
This is fine to ask here. I added discussions to keep support questions out of the issues section.

**Comment by motorstormm** [comment]
Hi,

Yes this a discussion😉

So I've had an answer from mayflash compagny ( dongle for switch) and it's easy to downgrade the Xbox one S controller even from the Xbox series X. This is the link :

  https://answers.microsoft.com/en-us/xbox/forum/all/revertrollbackdowngrade-xbox-controller-firmware/25f07169-3244-4fe0-838c-949951b7c109?auth=1

It's work well !

Have a nice day

**Comment by roice2000** [comment]
My Playstation 4 Dualshock Controller V1 does not work. 
But my other Playstation 4 Dualshock Controller V2 works fine.

Is the PS4 Controller V1 not supported ?? Its the one, where the Touchbar does not blink

**Reply by ndeadly** [reply]
The V1 controller, for whatever reason, can take a while before it will successfully pair with the console. If you hold the reset button on the controller and try again it often helps, but otherwise you just need to keep trying. Once paired it should connect without issue when you hit the PS button.

**Reply by roice2000** [reply]
thank you for the info. i will try it

**Comment by sandroddr** [comment]
In Atmosphère 1.2.6 work?

**Reply by ndeadly** [reply]
Yes

**Comment by joaovsoliveira** [comment]
There is a forecast for when it will be updated to suport the version 14.0.0?

**Reply by ndeadly** [reply]
When it's ready. If you can't wait there's a build of 0.6.4 recompiled for 14.0.0 here https://github.com/ndeadly/MissionControl/issues/439#issuecomment-1078589731

**Comment by impeeza** [comment]
you can watch the repo, or sear on gbatemp.

**Comment by Treydemark** [comment]
I got an error message when booting up my Switch that says

Error Desc: std: :abort() called (0xffe)

**Reply by ndeadly** [reply]
If you're on 14.0.0 this is because mission control needs to be updated to work on that firmware. I'm working on it. If you need it working now you can use this build https://github.com/ndeadly/MissionControl/issues/439#issuecomment-1078589731

**Comment by coldcoldsocold** [comment]
hi, is this version of missioncontrol compatible with 14.0.0? or should i wait until a compatible build drops?

**Reply by ndeadly** [reply]
No, need to wait for an update. See here for one that works with 14.0.0 in the meantime https://github.com/ndeadly/MissionControl/issues/439#issuecomment-1078589731

**Reply by coldcoldsocold** [reply]
oh ok, ill just wait, thanks!

**Reply by ndeadly** [reply]
FYI, the one posted fully supports 14.0.0 without compromise. I just have some new features I'd like to squeeze into the next major release that need finalising/testing.

**Reply by coldcoldsocold** [reply]
oh alright, thanks!

**Comment by HeroBoi01** [comment]
Hi, firstly thanks for this amazing software it's working like a charm!! I have a question, I successfully connect my official Wii u pro controller with the switch but I want to disable its rumble functionality. Is there any way to do that? I tried the missioncontrol.ini but it's for unofficial controller only. Thanks again !

**Reply by ndeadly** [reply]
Unofficial in this context means controllers that aren't supported by the switch out of the box. The Wii U Pro controller falls into this category. The .ini file should disable the rumble, if it isn't working then you've likely done something wrong. Common mistakes are not rebooting the console after making the changes, editing the template without copying/renaming as missioncontrol.ini, or not uncommenting (removing the leading `;`) the line in the ini you want to edit.

**Comment by SAG1002** [comment]
Hello, 
Could i connect my xbox series x/s controller ?
Thanks

**Reply by Masamune3210** [reply]
Pretty sure they use Bluetooth LE which is still not understood enough to be redirected like MC does, its being worked on slowly though

**Comment by HaruVM** [comment]
**controller Bluetooth issues and firmware downgrade on PC**

The new firmware works fine for me using the Xbox wireless dongle on PC but there are replies below from people using a Bluetooth connection to PC that lost functionality - for instance with an Elite controller

You can downgrade with a console but users have posted ways to revert on PC

One suggestion is " Open xbox accessories app then hit windows + R (run) and input this command:

xboxaccessories:\\firmwareupdate?legacyDowngrade=true

It should then give you option to revert."

**Reply by ndeadly** [reply]
Thanks for the info. Can you confirm that this works?

**Reply by HaruVM** [reply]
it really works, it dowgrades the version 5.13.3143.0 to 3.1.1221.0, i was unable to play stardewvalley with my wife because i use the gamepad for the pc and switch, now is all ok

**Reply by ndeadly** [reply]
Nice one, I'll add it to the readme.
